src/Car.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace MyStore;
+
+readonly class Car
+{
+    public string $name;
+    public int $year_made;
+    public float $zero_sixty;
+    public float $price;
+    public string $brand;
+
+    public function __construct($name, $year_made, $zero_sixty, $price, $brand)
```

Comment:
Even though the properties are type hinted, you should add type hints to the construct params as well